.github/workflows/pr-merged.yaml

```diff
@@ -23,17 +20,23 @@ jobs:
           github-token: ${{ secrets.NOTIFY_BOT_PAT_TOKEN }}
           script: |
             let branch, pr;
-            if (context.eventName == "pull_request_target") {
-              branch = context.payload.pull_request.base.ref;
-              pr = context.payload.pull_request.number;
-            } else if (context.eventName == "push") {
+            if (process.env.GITHUB_REF_TYPE === "tag") {
               const versionRegex = /v?(\d+)\.(\d+)\.\d+/;
               const versionMatch = process.env.GITHUB_REF_NAME.match(versionRegex);
               if (!versionMatch) {
                 return;
               }
               const [_, major, minor] = [...versionMatch];
               branch = `release-${major}.${minor}`;
+            } else if (process.env.GITHUB_REF_TYPE === "branch") {
+              branch = process.env.GITHUB_REF_NAME;
+              const commit = await github.rest.repos.getCommit({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                ref: process.env.GITHUB_SHA
+              });
+              const prMatch = commit.data.commit.message.match(/\(#(\d+)\)/);
+              pr = prMatch ? parseInt(prMatch[1]) : undefined;
```

Comment:
Do we even need the PR in the client payload?

Reply:
We don't!